modules/storage-api/src/testFixtures/java/org/apache/ignite/internal/storage/BaseMvStoragesTest.java

```diff
@@ -72,7 +74,7 @@ public abstract class BaseMvStoragesTest extends BaseIgniteAbstractTest {
             = MARSHALLER_FACTORY.create(SCHEMA_DESCRIPTOR, TestKey.class, TestValue.class);
 
     /** Hybrid clock to generate timestamps. */
-    protected final HybridClock clock = new HybridClockImpl();
+    protected static final HybridClock CLOCK = new HybridClockImpl();
```

Comment:
hm, good question, I don't really remember now. But is this change that bad?

Reply:
Statics are evil, especially if they store something with state. Clock's state is small now, but ideally it would be greate to have a fresh clock instance per test.

If it's too difficult to achieve, let's leave it static.

Reply:
ok, makes sense